src/coreComponents/constitutive/unitTests/testElasticTransverseIsotropic.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 Total, S.A
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2023-2024 Chevron
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+// Source includes
+#include "constitutive/ConstitutiveManager.hpp"
+#include "constitutive/solid/ElasticTransverseIsotropic.hpp"
+
+// TPL includes
+#include <gtest/gtest.h>
+#include <conduit.hpp>
+
+using namespace geos;
+using namespace ::geos::constitutive;
+
+TEST( ElasticTransverseIsotropicTests, testThermalExpansionAndTemperature )
+{
+  conduit::Node node;
+  dataRepository::Group rootGroup( "root", node );
+  ElasticTransverseIsotropic constitutiveModel( "model", &rootGroup );
+
+  // Create kernel updates
+  ElasticTransverseIsotropicUpdates updates = constitutiveModel.createKernelUpdates();
+
+  // Test that the default TEC derivative w.r.t. temperature and the default reference temperature are nil.
+  EXPECT_DOUBLE_EQ( updates.m_dThermalExpansionCoefficient_dTemperature, 0 );
+  EXPECT_DOUBLE_EQ( updates.m_referenceTemperature, 0 );
```

Comment:
do we need these tests? I mean having self-contained tests for the constitutive models is a good idea but it's unclear to me why you need to test this. Aren't you assigning this as default value?

Reply:
These unitests are added to pass one of the CI build that requires to cover a certain % of the code. Again, I am totally ok if you have another option for this that is more suitable to you.

Reply:
that CI test is not mandatory for merging.